orb/core/connection_types/sql/postgres/statements/create.py

```diff
@@ -24,6 +24,10 @@ def _createTable(self, model, owner, includeReferences):
             if not includeReferences and isinstance(col, orb.ReferenceColumn):
                 continue
 
+            # virtual flags do not exist in the database
+            elif col.testFlag(col.Flags.Virtual):
```

Comment:
however, they would still not exist in the database...if they exist in the db, they shouldn't be virtual.